src/igl/opengl/UniformAdapter.h

```diff
@@ -53,6 +53,7 @@ class UniformAdapter {
   std::vector<UniformState> uniforms_;
   std::vector<uint8_t> uniformData_;
   uint32_t maxUniforms_ = 1024;
+  size_t unformBufferAlignment_ = 16;
```

Comment:
Typo here, tut more importantly, this is only used inside an IGL_DEBUG_ASSERT so we don't need the variable at all, just fetch the value when needed.

Reply:
Typo has fixed.

Reply:
To query this value, we need to call the glGetIntegerv(GL_UNIFORM_BUFFER_OFFSET_ALIGNMENT, &tsize). If you query it during binding, it will result in additional OpenGL API calls per frame.

Reply:
like "uint32_t maxUniforms_ = 1024" above, it is used for check either.